duo_admin/plugin.spec.yaml

```diff
@@ -429,7 +457,80 @@ actions:
         title: Mintime
         type: integer
         description: Minimum time in UNIX timestamp milliseconds. Must be 13 or more digits in length
+        required: true
+        example: 1609377288936
+      maxtime:
+        title: Maxtime
+        type: integer
+        description: Maximum time in UNIX timestamp milliseconds. Must be 13 or more digits in length and greater than mintime. To use current time leave this parameter empty
+        required: false
+        example: 1611069760000
+      applications:
+        title: Applications
+        type: '[]string'
+        description: List of application IDs to filter on
+        required: false
+        example: ["DIV9C5V7T6L02DRWL4RU"]
+      users:
+        title: Users
+        type: '[]string'
+        description: List of user IDs to filter on
+        required: false
+        example: ["DUW2DKA44RFYECTU8R1O"]
+      event_types:
+        title: Event Types
+        type: '[]string'
+        description: List of event types(authentication, enrollment) to filter on, to include all leave this parameter empty
+        required: false
+        example: ["authentication"]
+      factors:
+        title: Factors
+        type: '[]string'
+        description: List of factors or methods used for an authentication attempt(duo_push, phone_call, u2f_token, hardware_token, bypass_code, sms_passcode, duo_mobile_passcode, yubikey_code, passcode, digipass_go_7_token, not_available, sms_refresh, remembered_device, trusted_network) to filter on, to include all leave this parameter empty
```

Comment:
We should remove the list of reasons here and point the user to the help documentation for the plugin to see all available inputs